R/labs.R

```diff
@@ -0,0 +1,103 @@
+#' @title Easily add ggplot labels using label attribtute of `data.frame` column
+#' @description Applies same logic as \code{\link[ggplot2]{labs}} but uses as default
+#' the column label attribute if present as the variable label in the plot.
+#' @param ... A list of new name-value pairs. The name should either be an aesthetic, 
+#' or one of "title", "subtitle", or "caption"
+#' @return NULL
+#' @examples 
+#' 
+#' require(ggplot2)
+#' iris_labs <- iris
+#' lbl <- c('Sepal Length', 'Sepal Width', 'Petal Length', 'Petal Width', 'Flower Species')
+#' labelled::var_label(iris_labs) <- split(lbl,names(iris_labs))
+#' 
+#' p <- ggplot(iris_labs,aes(x=Sepal.Length,y=Sepal.Width))+
+#'   geom_line(aes(colour=Species))
+#'   
+#' p
+#' p + easy_labs()
+#' p + easy_labs(subtitle='mysubtitle', x='x axis label')
+#'  
+#' @rdname easy_labs
+#' @export 
+easy_labs <- function(...){
+
+  man_labs <- labs(list(...))
```

Comment:
Do we need to explicitly namespace prefix or explicitly `importFrom`? `ggplot2::labs()`